Dockerfile

```diff
@@ -4,20 +4,36 @@ FROM bash:${bashver}
 ARG TINI_VERSION=v0.19.0
 ARG TARGETPLATFORM
 
+# https://github.com/opencontainers/image-spec/blob/main/annotations.md
+LABEL maintainer="Bats-core Team"
+LABEL org.opencontainers.image.authors="Bats-core Team"
+LABEL org.opencontainers.image.title="Bats"
+LABEL org.opencontainers.image.description="Bash Automated Testing System"
+LABEL org.opencontainers.image.url="https://hub.docker.com/r/bats/bats"
+LABEL org.opencontainers.image.source="https://github.com/bats-core/bats-core"
+LABEL org.opencontainers.image.base.name="docker.io/bash"
+
 COPY ./docker /tmp/docker
 # default to amd64 when not running in buildx environment that provides target platform
 RUN /tmp/docker/install_tini.sh "${TARGETPLATFORM-linux/amd64}"
-    
+# Install bats libs
+RUN /tmp/docker/install_libs.sh support 0.3.0
```

Comment:
@martin-schulze-vireso  do you prefer the version hardcoded here or as args? (4 args..)

Reply:
I think we should have args but rely on defaults that are updated with each release. This allows for customization but a progressing default.

Reply:
done